### PR TITLE
Cloud Run-based agent for running the validator on the googleapis repo.

### PR DESCRIPTION
**Pull request by timburks**
Results are stored in a Google Sheet.

### CODE REVIEW

**Comment by timburks** [comment]
This is messy (and not ready to merge) but seems to work. 

When the app is uploaded to Cloud Run, the service returns a URL for the agent that looks like this: https://config-checker-abcd1234fg-hi.a.run.app. The following `curl` command triggers the checker to run against `googleapis` and put the results in a spreadsheet:
```
curl  https://config-checker-abcd1234fg-hi.a.run.app/check
```

**Comment by noahdietz** [comment]
You've said `This is messy (and not ready to merge) but seems to work.`. Is there something else you'd like to do?

File: agent/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Build the command inside the container.
+# (You may fetch or manage dependencies here,
+# either manually or with a tool like "godep".)
+RUN CGO_ENABLED=0 GOOS=linux go get 
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o config-checker
+
+ENV PATH="/usr/local/bin:${PATH}"
+
+ENV GOOGLE_APPLICATION_CREDENTIALS "/go/src/github.com/googleapis/config-checker/credentials.json"
```
Comment (noahdietz) [inline agent/Dockerfile]: If possible, could we remove this and instead deploy the app with a [managed service account]? Does the client library work with the application default credentials?

`gcloud beta run deploy ... --service-account SERVICE_ACCOUNT_EMAIL` 

[managed service account]: https://cloud.google.com/run/docs/securing/service-identity

File: agent/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+build:
+	gcloud builds submit --tag gcr.io/config-checker/config-checker
```
Comment (noahdietz) [inline agent/Makefile]: please add `--project=config-checker`

File: agent/Makefile [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+deploy:
+	gcloud beta run deploy --image gcr.io/config-checker/config-checker --platform managed
```
Comment (noahdietz) [inline agent/Makefile]: please add `--project=config-checker --region=us-central1`

**Comment by noahdietz** [comment]
Closing this PR as it is no longer necessary with the migration done.

**Pull request closed without merging**